.github/ISSUE_TEMPLATE/3_maintenance_issue.yml

```diff
@@ -0,0 +1,22 @@
+name: 🏗️ Maintenance
+description: A change in the code to overcome technical debt
+labels: ["t:maintenance"]
+assignees: ["pcrespov"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues
+          required: true
+  - type: textarea
+    attributes:
+      label: Planned changes
+      description: |
+        After describing the changes, please connect tohttps://github.com/ITISFoundation/osparc-issues/issues/675
+
+        Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+      value: Please connect this case to the Epic https://github.com/ITISFoundation/osparc-issues/issues/675

```

Comment:
maybe it can also be automatically related to it? so at least there will be an entry.

Reply:
NO, unfortunately no. I will try that in a second round where these PRs could be created via the API